### ISSUE TITLE
familiarize yourself with hackmd for collaborative work

### ISSUE THREAD

**Issue by NiklasTR** [labdao/newsletter#1]

**Comment by NiklasTR** [comment]
* I think github is the perfect way to show the world what we are working on and give everybody a chance to observe and contribute meaningfully within the community - irrespective of wether they happen to be in the discord chat for the writing team or not.
* I recommend creating a hackmd.io account for collaborative editing and sharing it with trusted collaborators via internal communication tools

**Comment by tylerstahl** [comment]
I agree, I will look into hackmd.io and set up an account. What role do you see publishing via hackmd (internal purposes?) vs publishing to our website (external purposes?).

**Comment by bjdyakov** [comment]
Great points from Tyler. I'm behind anything that lets us more transparently share work and that enables us to collaborate more effectively.

I made an account on HackMD but I think working on an actual note and trying to push it to Github is going to be important for me to really learn how/why to use this tool.

+1 to writing some clear documentation on how to navigate and contribute through our stack (Discord, Dework, Github, HackMD... and then pushing it all to our website? Or maybe Gitbook?) (note: I don't know how Gitbook works either, but I've seen several projects use it as their publicly-viewable "wiki"/ knowledge base and like how it looks. Presumably it integrates nicely with all of the other tools I just mentioned)

I think the key thing is that it's **very** easy for people to access and read the info, and _fairly_ easy to learn how to add info (maybe 30-60 minutes reading some intro documentation, making accounts, and learning... which is minimal if someone is serious about contributing to the project)

**Comment by hossam-zaki** [comment]
Think it'd be good to figure out how to go through our stack, kind of confusing as of now and am getting lost between things. 

Can we directly integrate hackmd into this repo?
